src/integration/kotlin/pl/edu/agh/gem/integration/BaseIntegrationSpec.kt

```diff
@@ -16,6 +17,8 @@ import pl.edu.agh.gem.integration.environment.ProjectConfig
 )
 @ActiveProfiles("integration")
 abstract class BaseIntegrationSpec(body: ShouldSpec.() -> Unit) : ShouldSpec(body) {
+
+    override fun testCaseOrder(): TestCaseOrder? = TestCaseOrder.Sequential
```

Comment:
why we added that ? could you import that directly ?

Reply:
didn't figured out how to reset mocked email server for each test so i had to determine order of tests so i can do the assertions

Reply:
you could restart each time using 'AfterTestListener'  interface